github-issue-get-eta.go

```diff
@@ -0,0 +1,83 @@
+/*
+ * Buzz, (C) 2016,2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+	"strconv"
+	"encoding/json"
+	"strings"
+)
+
+type eta_struct struct {
```

Comment:
Could we change this to [ETAStruct](https://golang.org/doc/effective_go.html#mixed-caps) or similar?